example/README.md

```diff
@@ -0,0 +1,66 @@
+Example for creating an custom Lambda Layer to run PHP 7
```

Comment:
It would be good not to have the instructions for the example in two places (main README and example README). What do you think of linking back to the relevant section of the main README here, or vice versa linking to this example README from the main README?

Reply:
Makes sense to me.
I have removed the child readme.